src/memory_model.ts

```diff
@@ -624,7 +630,7 @@ export class MemoryModel {
                     style.text_value["fill"] = this.text_color;
                 }
                 if (!style.text_value.hasOwnProperty("text-anchor")) {
-                    style.text_value["text-anchor"] = "begin";
+                    style.text_value["text-anchor"] = "start";
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/text-anchor#usage_notes `begin` isn't an option